pkg/diagnose/scan.go

```diff
@@ -165,31 +165,6 @@ func Scan(ctx context.Context, opts ...OpOption) error {
 					} else {
 						log.Logger.Warnw("failed to check gpm supported or not", "error", err)
 					}
-
-					if gpmSupported {
```

Comment:
if we delete the code, we'd remove all related code? from L158  to L167

Reply:
@eahydra Yes, this PR already deletes `RecvGPMEvents` and such.